### DISCUSSION TITLE
Datadog Racecar, DeliveryBoy, Kafka?

### DISCUSSION THREAD

**Discussion by henrikbjorn** [zendesk/racecar#273] [Q&A] [answered]
It seems that each for the libraries have their own Datadog integration that uses ActiveSupport Notifications for measurering.

The one in Racecar and Kafka seems to be almost identical. I am worried that enabling both would skew the numbers.

Maybe Racecar should just use the one from the Kafka library ?

**Comment by dasch** [comment, marked as answer]
Racecar v1 uses the integration from ruby-kafka already. You should feel free to enable both.
